From db602b3b3328275359e5714df33ae449e6e93774 Mon Sep 17 00:00:00 2001
From: Michael Natterer <mitch@gimp.org>
Date: Fri, 20 Dec 2013 12:11:53 +0000
Subject: Bug 720492 - Letter spacing in Text Tool not reset to default

Also reset the "letter-spacing" property in gimp_text_options_reset().

(cherry picked from commit 81c02e7db1fa96ec6b7f0e84d4bc8b9ce2dca293)
---
diff --git a/app/tools/gimptextoptions.c b/app/tools/gimptextoptions.c
index 324fa13..061e1fb 100644
--- app/tools/gimptextoptions.c
+++ app/tools/gimptextoptions.c
@@ -380,6 +380,7 @@ gimp_text_options_reset (GimpToolOptions *tool_options)
   gimp_config_reset_property (object, "justify");
   gimp_config_reset_property (object, "indent");
   gimp_config_reset_property (object, "line-spacing");
+  gimp_config_reset_property (object, "letter-spacing");
   gimp_config_reset_property (object, "box-mode");
   gimp_config_reset_property (object, "use-editor");
 }
--
cgit v0.9.2
